sample/benchmark/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
In this case we should remove this file

Reply:
I removed it, if we needed it later we can add it back